docs/USAGE.md

```diff
@@ -151,6 +151,17 @@ the BindingRedirect sample can only run successfully under the instrumentation s
 binding redirect makes the application dependent on a version of `System.Diagnostics.DiagnosticSource`
 that is not available during building time.
 
+#### Additional-Deps and Runtime Package Store
```

Comment:
shouldn't we clearly say that it is ONE of the possibilities to fix the issue?
The next section `.NET Core Dependency File` is also about resolving assembly conflicts. The users may think that they need to follow both instructions

Reply:
Realized information was lost during the commit, will get it updated.